clients/drcachesim/common/options.cpp

```diff
@@ -268,8 +268,16 @@ droption_t<bytesize_t> op_warmup_refs
 (DROPTION_SCOPE_FRONTEND, "warmup_refs", 0,
  "Number of memory references to warm caches up",
  "Specifies the number of memory references to warm up caches before simulation. "
- "The warmup references come after the skipped references "
- "and before the simulated references.");
+ "The warmup references come after the skipped references and before the "
+ "simulated references. This flag is incompatible with warmup_fraction.");
+
+droption_t<double> op_warmup_fraction
+(DROPTION_SCOPE_FRONTEND, "warmup_fraction", 0.0, 0.0, 1.0,
```

Comment:
It may be worth adding a line for this feature to the changelist https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L200

Reply:
Done.